### DISCUSSION TITLE
Support of Non Profit Organizations

### DISCUSSION THREAD

**Discussion by lemaadi** [vercel/community#823] [Idea]
Hi Community,

I would like to know if you are supporting `Non Profit Organizations` and offering a special plan 

Thank you

**Comment by amyegan** [comment]
Hi @lemaadi! Vercel does support a number of open source, educational, and charitable projects. You can find an outline of sponsorship criteria and contact information at [vercel.com/guides/can-vercel-sponsor-my-open-source-project](https://vercel.com/guides/can-vercel-sponsor-my-open-source-project)

**Reply by lemaadi** [reply]
Thank you @amyegan :heart:

**Comment by tyteen4a03** [comment]
Hi, I'm currently considering a few options for a non-profit I'm working with. The guide only covers open source projects. Are there any guidance for nonprofits?

**Reply by amyegan** [reply]
Hey @tyteen4a03! Our free tier can handle most small projects, but larger projects or those involving [commercial usage](https://vercel.com/docs/limits/fair-use-policy#commercial-usage) would need to upgrade to a Pro plan.  

If you [reach out to our Sponsorship team](https://vercel.com/guides/can-vercel-sponsor-my-open-source-project#applying-for-sponsorship) and share some more info about your project, they can help you determine if it qualifies for sponsorship. Be sure to explain why Vercel's free tier is not enough for your project and include your Vercel team slug (if you already have one) in the email. 🙂

**Reply by nikos-pblworks** [reply]
Hey Amy. The "reach out" link doesn't seem to have any form or something similar to contact the Sponsorship team. Is there and e-mail we can use to react out to them?

**Reply by amyegan** [reply]
Thanks for reaching out, @nikos-pblworks. We are not currently accepting new team sponsorships, but individuals can still deploy non-commercial projects for free within the the [free tier usage guidelines](https://vercel.com/docs/limits/fair-use-guidelines#typical-monthly-usage-guidelines).

**Reply by wingedfox** [reply]
When it might be continued?
